tests/system/model_monitoring/test_app.py

```diff
@@ -212,6 +223,21 @@ def custom_setup_class(cls) -> None:
                     "with_training_set": True,
                 },
                 results={"data_drift_test"},
+            ),  # TODO : delete in 1.9.0
+            _AppData(
+                class_=DemoMonitoringAppV2,
+                rel_path="assets/application.py",
+                results={"data_drift_test", "model_perf"},
+            ),
```

Comment:
Don't you think we have too many apps for this test with V1 + V2?

Reply:
Because we still need to support V1 until 1.9.0 I think we should test V1 as well.. what is your suggestion? To split it to V1 and V2 tests?

Reply:
Ok, testing also V1 is a valid point.
I think you can make this class slightly more abstract and split V1 from V2, without duplicating the class